src/helpers/classNamesToString.ts

```diff
@@ -5,6 +5,9 @@ type AllowedExtraClassnames =
   | 'stroke-text-small'
   | 'retro-text'
   | '-my-12'
+  | 'outlined-button'
+  // built-in tailwind classname, not represented as helper in tailwindcss-classnames for some reasons
```

Comment:
It's pretty obvious if we'll look into the source files, so let's remove the comment

Reply:
Fixed